### PR TITLE
Update README to support all clients.

### PR DESCRIPTION
**Pull request by LaContra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,17 @@
...
 
-## Install Prysm beacon node and validator with Helm
+## Detailed guide for running clients on Prater
 
-If the goal is to run Prysm on mainnet, we recommend to use
+We've written [a blog post](https://lumostone.com/en/eth2-staking-with-k8s-prysm/) detailing the requirement and walkthrough for running Prysm on Prater. We'll add guides for other clients soon!
+
+## Install beacon node and validator with Helm
+
+If the goal is to run Ethereum 2.0 clients on mainnet, we recommend to use
```
Comment (chingyi-lin) [inline README.md]:
```suggestion
If the goal is to run Ethereum 2.0 clients on mainnet, we recommend:
```

File: README.md [hunk 2]
```diff
@@ -2,13 +2,17 @@
...
 
 **This project is still in development and it's NOT recommended to be used on mainnet.**
 
-## Install Prysm beacon node and validator with Helm
+## Detailed guide for running clients on Prater
 
-If the goal is to run Prysm on mainnet, we recommend to use
+We've written [a blog post](https://lumostone.com/en/eth2-staking-with-k8s-prysm/) detailing the requirement and walkthrough for running Prysm on Prater. We'll add guides for other clients soon!
+
+## Install beacon node and validator with Helm
```
Comment (chingyi-lin) [inline README.md]:
```suggestion
## Quick start: Install beacon node and validator with Helm
```

File: README.md [hunk 3]
```diff
@@ -37,70 +41,108 @@ If the goal is to run Prysm on mainnet, we recommend to use
...
-   We recommend checking each field in `values.yaml` to determine the desired configuration. Fields that need to be changed or verified before installing the chart are the following ones:
-
-   - **nfs.serverIp**: NFS server IP address.
-   - **image.version**: Prysm client version.
-   - **beacon.dataVolumePath**: The path to the data directory on the NFS for the beacon node.
-   - **beacon.web3Provider** and **beacon.fallbackWeb3Providers**: Ethereum 1 node endpoints.
-   - **validatorClients.validatorClient1.dataVolumePath**: The path to the data directory on the NFS for the validator.
-   - **validatorClients.validatorClient1.walletVolumePath**: The path to the data directory on the NFS for the wallet.
-   - **validatorClients.validatorClient1.walletPassword**: The wallet password.
+2. Change values in the target client's values.yaml. For example, `./prysm/helm/values.yaml`.
```
Comment (chingyi-lin) [inline README.md]:
```suggestion
2. Change values in the target client's values.yaml. For example, change values in `./prysm/helm/values.yaml` for prysm client.
```

File: README_hostpath.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+    ```bash
+    kind create cluster --config=cluster-config/kind-single-node.yaml 
+    ```
+
+2. Create namespace. Remember to change `<namespace>` to the same namespace value as in the manifests. 
+
+    ```bash
+    kubectl create namespace <namespace>
+
```
Comment (chingyi-lin) [inline README_hostpath.md]: missing ` ``` `?

File: README_nfs.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+7. Review all manifests and change the values if needed.
+
+## Config the cluster
+
+1. Create namespace. Remember to change `<namespace>` to the same namespace value as in the manifests. 
+
+    ```bash
+    kubectl create namespace <namespace>
+
```
Comment (chingyi-lin) [inline README_nfs.md]: missing ```?

File: README_hostpath.md (hunk 4 above)
Comment (LaContra) [reply README_hostpath.md]: Good catch! Added.

File: README_nfs.md (hunk 5 above)
Comment (LaContra) [reply README_nfs.md]: Added.

**Review by chingyi-lin — APPROVED** [review]

**Pull request merged**